apps/web/src/design-system/config/colors.ts

```diff
@@ -18,7 +18,7 @@ export const colors = {
   gradientStart,
   gradientEnd,
   success: '#4D9980',
-  warning: '#FF8000',
+  warning: '#EAA900',
```

Comment:
Not sure we can replace it here can you pass it a prop instead and make the "FF8000" as default color prop?

Reply:
I have already passed it as a prop in my previous commit.

```
<WarningIcon color={color} onClick={onClick} />;

export function WarningIcon(props: React.ComponentPropsWithoutRef<'svg'>) {
  return (
    <svg width="18" height="15" viewBox="0 0 18 15" fill="none" xmlns="http://www.w3.org/2000/svg" {...props}>
      <path
        d="M16.8125 13.0312L10.125 1.65625C9.625 0.78125 8.34375 0.78125 7.8125 1.65625L1.15625 13.0312C0.65625 13.9062 1.28125 15 2.3125 15H15.6562C16.6875 15 17.3125 13.9062 16.8125 13.0312ZM8.25 5.25C8.25 4.84375 8.5625 4.5 9 4.5C9.40625 4.5 9.75 4.84375 9.75 5.25V9.25C9.75 9.6875 9.40625 10 9 10C8.5625 10 8.25 9.6875 8.25 9.25V5.25ZM9 13C8.4375 13 8 12.5625 8 12.0312C8 11.5 8.4375 11.0625 9 11.0625C9.53125 11.0625 9.96875 11.5 9.96875 12.0312C9.96875 12.5625 9.53125 13 9 13Z"
        fill="currentColor"
      />
    </svg>
  );
```

Is this what you mean?

Reply:
Ah yes but lets not change the color in colors but hard coded it for now where you need it.

Reply:
Made the changes as required.